### PR TITLE
Task-59855: Update on Mobile_iOS_jitsi_join a call_US01 task

### PR DESCRIPTION
**Pull request by wbenabdessalem**
To manage the pop-up functionality, We added the Install button in the pop-up window to redirect the user to the Jitsi app in the App Store.

### CODE REVIEW

File: eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@ import UIKit
 enum ActionHandler {
     case defaultAction
     case delete
+    case store
```
Comment (ahamdi) [inline eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift:13]: the word "store" refers to what ?

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift (hunk 1 above)
Comment (wbenabdessalem) [reply eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift:13]: To manage the pop-up functionality, in fact, Case Store adds the Install button in the pop-up window to redirect the user to the Jitsi app on the App Store.

Comment (ahamdi) [reply eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift:13]: Ok fine
Could you update the description of the PR ?

Comment (wbenabdessalem) [reply eXo/Sources/Controllers/AddDomainVC/CustomPopupVC/CustomPopupViewController.swift:13]: Done

**Pull request closed without merging**